lib/rules/focus.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "focus",
+  "selector": "a[href][onFocus], link[href][onFocus], button[onFocus], input[onFocus], select[onFocus], textarea[onFocus], audio[controls][onFocus], video[controls][onFocus]",
```

Comment:
Minor point, but `link` isn't a clickable element. `area` and `summary` are though. Should switch that up.

Reply:
For some reason, do the selector have an problem with `area` element. It works just fine in Chrome, however Firefox and circleci tests cant find the element, and reports an missing element in the tests. Have yet to figure out why these two browsers differs on this element.